<launch>
	<arg name="uav_id" default="1"/>
	<arg name="cen_flag" default="0"/>
	<arg name="uav_size" default="1"/>
	<arg name="swarm_id" default="1"/>
	<node pkg="decision_maker" type="each_uav_h.py" name="uav_node_$(arg uav_id)" output="screen">
		<param name="uav_id" value="$(arg uav_id)"/>
		<param name="cen_flag" value="$(arg cen_flag)"/>
		<param name="uav_size" value="$(arg uav_size)"/>
		<param name="swarm_id" value="$(arg swarm_id)"/>
	</node>
</launch>

